source/help/apps/desktop-guide.rst

```diff
@@ -0,0 +1,187 @@
+===================================
+Desktop Application's End User's Guide
```

Comment:
Thinking it could just be "User's Guide," since End User is not a term most people are familiar with outside the dev/design process, I think.